compiler/rustc_feature/src/lib.rs

```diff
@@ -112,12 +106,19 @@ fn find_lang_feature_issue(feature: Symbol) -> Option<NonZeroU32> {
             .chain(STABLE_REMOVED_FEATURES)
             .find(|t| t.name == feature);
         match found {
-            Some(found) => found.issue(),
+            Some(found) => found.issue,
             None => panic!("feature `{}` is not declared anywhere", feature),
         }
     }
 }
 
+const fn to_nonzero(n: Option<u32>) -> Option<NonZeroU32> {
+    match n {
+        None => None,
+        Some(n) => NonZeroU32::new(n),
```

Comment:
This is a trivial wrapper over `n.and_then(NonZeroU32::new)`, I don't think it's necessary.

Reply:
Sure, but `and_then` isn't `const`.

Reply:
Could you add another FIXME to remove `fn to_nonzero` then?

Reply:
Or actually extend the old one, trading one FIXME for another doesn't look like an improvement, just some churn.